js/analyzer.js

```diff
@@ -113,25 +117,53 @@ Analyzer.prototype.subheaderChecker = function() {
     return result;
 };
 
+/**
+ * check if the keyword contains stopwords
+ */
+Analyzer.prototype.stopwordChecker = function(){
+    matches = this.stringCounter(this.config.keyword, this.config.stopWords);
+    result = {name: 'stopWords', result: {count: matches.length, matches: matches}, rating:5 };
+    return result;
+}
+
 /**helper functions*/
 
 /**
- * removes certain words from string
- * @params textString
- * @returns textString without keywords
- */
-Analyzer.prototype.stripKeywords = function(textString){
-    //words to remove
-    var wordString = '';
-    for (var i = 0; i < this.config.wordsToRemove.length; i++){
-        if(wordString.length > 0){ wordString += '|'; }
-        wordString += '('+this.config.wordsToRemove[i]+')\\b';
+ * removes strings from array and replaces them with keyword.
+ * @param textString
+ * @param stringsToRemove []
+ * @param replacement (default == space)
+ * @returns {textString}
+ */
+Analyzer.prototype.stringReplacer = function(textString, stringsToRemove, replacement){
+    if(typeof replacement == 'undefined'){replacement = ' '};
+    textString = textString.replace(this.regexStringBuilder(stringsToRemove), replacement);
+    return yst_pp.stripSpaces(textString);
+}
+
+/**
+ * matches string with given array of strings to match.
+ * @param textString
+ * @param stringsToMatch
+ * @returns {matches}
+ */
+Analyzer.prototype.stringCounter = function(textString, stringsToMatch){
```

Comment:
This method doesn't really count anything. It matches an array of strings to a string. Please rename it to `matchStrings` or `matchStringsArray`.